common/amazon.ts

```diff
@@ -61,8 +61,10 @@ export async function category(url: string) {
   const tld = url.split('amazon.')[1].split('/')[0]
   const path = url.split(tld + '/')[1].split('?')[0]
 
+  console.log(`https://www.amazon.${tld}/${path}/?ie=${ie}&node=${node}`);
```

Comment:
This seems like a leftover that doesn't need to be there anymore

Reply:
Fixed.